attestation-service/src/bin/restful-as.rs

```diff
@@ -75,6 +82,67 @@ pub enum RestfulError {
     Anyhow(#[from] anyhow::Error),
 }
 
+fn configure_cors(allowed_regions: &[String]) -> Cors {
+
+    info!("Raw allowed_regions input: {allowed_regions:?}");
+
+    // 1. Set Default region
+    let default_allowed = vec!["http://localhost:3000".to_string()];
+
+    // 2. Set regions
+    let regions = if !allowed_regions.is_empty() {
+
+        let expanded_regions: Vec<String> = allowed_regions
+            .iter()
+            .flat_map(|s| s.split(','))
+            .map(|s| s.trim().to_string())
+            .filter(|s| !s.is_empty())
+            .collect();
+
+        if expanded_regions.is_empty() {
+            log::warn!("No valid regions found after processing input");
+            default_allowed
+        } else {
+            expanded_regions
+        }
+    } else {
+        default_allowed
+    };
+
+    log::info!("Processed regions: {:?}", regions);
+
+    // 3. Configure CORS
+    let mut cors = Cors::default()
+        .allowed_methods(vec!["POST", "GET", "OPTIONS"])
+        .allowed_headers(vec![header::CONTENT_TYPE, header::AUTHORIZATION])
+        .max_age(86400);
+
+    // 4. Add allowed region
+    for region in &regions {
+        log::debug!("Adding CORS allowed origin: {}", region);
+        if region.starts_with("http://") || region.starts_with("https://") {
+            log::debug!("Adding valid CORS origin: {}", region);
+            cors = cors.allowed_origin(region.as_str());
+        } else {
+            log::error!("Invalid CORS origin format: '{}'. Must start with http:// or https://", region);
+        }
+    }
+
+    // 5. Check region status
+    if regions.is_empty() {
+        #[cfg(debug_assertions)]
+        log::warn!("Using default CORS origin: http://localhost:3000");
+
+        #[cfg(not(debug_assertions))]
+        log::warn!("No CORS origins specified - blocking all cross-origin access");
+    } else {
+        log::info!("Configured CORS for {} allowed origin(s)", regions.len());
```

Comment:
looks like we do not print any `info!` level log for CORS origins. Probably we can do this here
```suggestion
        info!("Configured allowed CORS origin(s): {regions:?}");
```